sdk/eventgrid/Azure.Messaging.EventGrid/src/Customization/EventGridModelFactory.cs

```diff
@@ -5,11 +5,13 @@
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Text.Json;
+using Azure.Core;
 using Azure.Messaging.EventGrid.SystemEvents;
 
 namespace Azure.Messaging.EventGrid
 {
 #pragma warning disable CA1054 // URI-like parameters should not be strings
+    [CodeGenType(nameof(EventGridModelFactory))]
```

Comment:
What is the impact of this change?

Reply:
This prevents `EventGridModelFactory` from being generated in `Models` namespace cause `EventGridModelFactory` has been GA'd